src/community/advanced-data-card.tsx

```diff
@@ -83,23 +82,12 @@ const CardContent: React.FC<CardContentProps> = ({
     subtitleLinesMax,
     description,
     descriptionLinesMax,
-    ariaLabel,
-    ...touchableProps
 }) => {
     const {textPresets} = useTheme();
 
     return (
         <Stack space={4}>
             {headline}
-            <Touchable
```

Comment:
this empty touchable was inaccessible. Screen readers can't read the card content